src/main/scala-2.13/org/apache/james/gatling/smtp/BaseActor.scala

```diff
@@ -0,0 +1,28 @@
+package org.apache.james.gatling.smtp
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.Duration
+
+import akka.actor.{ Actor, ActorSystem, Scheduler, Terminated }
+import com.typesafe.scalalogging.LazyLogging
+
+/**
+ * Cloned from io.gatling.core.akka.BaseActor
+ */
+abstract class BaseActor extends Actor with LazyLogging {
```

Comment:
You cloned that on gatling-imap too no? Can't just retake it from there instead of copying it again?